.eslintrc.json

```diff
@@ -0,0 +1,27 @@
+{
+  "env": {
+    "node": true,
+    "commonjs": true,
+    "es2021": true
```

Comment:
Entered this because nodejs 14 is the default on GitHub Ubuntu and has support for this. Just found that the action uses nodejs 12: (https://github.com/Pararius/action-terraform/blob/pre-merge/action.yaml#L24-L26)
```yaml
runs:
  using: node12
  main: dist/index.js
```

Should we update to nodejs 14 (or 16) as well? Both are LTS releases.